cluster/gce/config-default.sh

```diff
@@ -249,6 +249,13 @@ ENABLE_DEFAULT_STORAGE_CLASS="${ENABLE_DEFAULT_STORAGE_CLASS:-true}"
 # Optional: Enable legacy ABAC policy that makes all service accounts superusers.
 ENABLE_LEGACY_ABAC="${ENABLE_LEGACY_ABAC:-true}" # true, false
 
+# Optional: use a default, fairly open PodSecurityPolicy
+ENABLE_POD_SECURITY_POLICY="${ENABLE_POD_SECURITY_POLICY:-true}"
```

Comment:
nit: I'd put it just below definition of `ADMISSION_CONTROL` to make it obvious that it's not empty.